controller/config.go

```diff
@@ -273,6 +288,21 @@ func (c Config) AuditLogMaxBackups() int {
 	return DefaultAuditLogMaxBackups
 }
 
+// AuditLogExcludeMethods returns the set of method names that are
+// considered uninteresting for audit logging. Conversations
+// containing only these will be excluded from the audit log.
+func (c Config) AuditLogExcludeMethods() set.Strings {
+	if value, ok := c[AuditLogExcludeMethods]; ok {
+		value := value.([]interface{})
```

Comment:
If the schema is schema.List(schema.String()), what isn't value []string?
Is there a coerce call missing somewhere?

Reply:
The coerce call ensures the value is really a string slice, but it can't tell the type system - it still comes out as a `[]interface{}` and you need to convert it as far as I can tell. 

Digging into it, it's because `juju/schema.List.Coerce` constructs a `[]interface{}` for its output. It doesn't know what type its child `Checker` will return for elements. We could add a special-case `StringList` that returned a `[]string` instead, or a special case for common child types in List I guess.